docs/server/source/appendices/run-with-docker.md

```diff
@@ -30,45 +38,84 @@ Ready to go!
 Let's analyze that command:
 
 * `docker run` tells Docker to run some image
+* `--interactive` keep STDIN open even if not attached
 * `--rm` remove the container once we are done
-* `-v "$HOME/bigchaindb_docker:/data"` map the host directory
+* `--tty` allocate a pseudo-TTY
+* `--volume "$HOME/bigchaindb_docker:/data"` map the host directory
  `$HOME/bigchaindb_docker` to the container directory `/data`;
  this allows us to have the data persisted on the host machine,
  you can read more in the [official Docker
  documentation](https://docs.docker.com/engine/tutorials/dockervolumes/#/mount-a-host-directory-as-a-data-volume)
-* `-t` allocate a pseudo-TTY
-* `-i` keep STDIN open even if not attached
-* `bigchaindb/bigchaindb` the image to use
-* `-y configure` execute the `configure` sub-command (of the `bigchaindb` command) inside the container, with the `-y` option to automatically use all the default config values
+* `bigchaindb/bigchaindb` the image to use. All the options after the container name are passed on to the entrypoint inside the container.
+* `-y configure` execute the `configure` sub-command (of the `bigchaindb`
+ command) inside the container, with the `-y` option to automatically use all the default config values
+* `--dev-allow-temp-keypair` specifies that this is a dev environment and
+  enables creation of temporary key pairs to be written to the configuration
+  file
+* `mongodb` or `rethinkdb` specifies the database backend to use with bigchaindb
+
+
+### Run the backend database
+From v0.9 onwards, you can run either RethinkDB or MongoDB.
+
+We use the virtual interface created by the docker daemon to allow
+communication between the BigchainDB and database containers.
+It has an IP address of 172.17.0.1 by default.
```

Comment:
We will not use the docker vnic in production, as both containers need routable IPs.
From [here](https://docs.docker.com/engine/userguide/networking/):
`
The bridge network represents the docker0 network present in all Docker installations. Unless you specify otherwise with the docker run --network=<NETWORK> option, the Docker daemon connects containers to this network by default. You can see this bridge as part of a host’s network stack by using the ifconfig command on the host.
`